vscode/microsoft-kiota/src/extension.ts

```diff
@@ -282,6 +282,7 @@ export async function activate(
         }));
         if (config.descriptionPath) {
           await openTreeViewWithProgress(() => openApiTreeProvider.setDescriptionUrl(config.descriptionPath!));
+          void vscode.window.showInformationMessage(vscode.l10n.t('You can now select the required endpoints from ') + openApiTreeProvider.apiTitle);
```

Comment:
The positioning of this, means if an error is encountered in loading the tree when calling openApiTreeProvider.setDescriptionUrl for example, the message to select required endpoints would still show?

Reply:
Nice catch!